nncf/quantization/algorithms/fast_bias_correction/backend.py

```diff
@@ -104,11 +105,12 @@ def mean_statistic_collector(reduction_shape: ReductionShape,
 
     @staticmethod
     @abstractmethod
-    def get_tensor_names(node: NNCFNode) -> Tuple[List[str], List[str]]:
+    def get_input_output_names(node: NNCFNode, nncf_graph: NNCFGraph) -> Tuple[List[str], List[str]]:
```

Comment:
What is the correct type of return value?
`Tuple[str, str]`
or
`Tuple[List[str], List[str]]`